src/index.ts

```diff
@@ -7,5 +7,10 @@ import server from './server';
 
 // **** Start server **** //
 
-const msg = ('Express server started on port: ' + EnvVars.port.toString());
-server.listen(EnvVars.port, () => logger.info(msg));
+const msg = (port: number | string) => (
+  'Express server started on port: ' + port.toString()
+);
+
+server.listen(EnvVars.ports[0], () => logger.info(msg(EnvVars.ports[0])));

```

Comment:
why exactly are we starting the server on 3 ports?

Reply:
Check Madhavan comment above.